frontend/sso/routes.py

```diff
@@ -15,13 +15,19 @@
 @sso_bp.route("/signed-out/<status>")
 def signed_out(status):
     return_app = request.args.get("return_app")
+    kwargs = (
+        {"service_title": Config.SAFE_RETURN_APPS[return_app].service_title}
+        if return_app
+        else {}
+    )
     return (
         render_template(
             "sso_signed_out.html",
             status=status,
             login_url=url_for(
                 Config.SSO_LOGIN_ENDPOINT, return_app=return_app
             ),
+            **kwargs,
```

Comment:
Any reason for exploding a dictionary here rather than just using `service_title` explicitly? 

I'd maybe avoid using `kwargs` as a variable name since by convention it's typically used as a parameter name, but that's a nit-pick so feel free to ignore i.e.:

```python
def func(*args, **kwargs): ...
```

Also - I'm guessing this overrides the `service_title` that's injected globally?

https://github.com/communitiesuk/funding-service-design-authenticator/blob/eb444f0e81d3e1d8b2bb68a58a02c1679143ebbb/app.py#L129-L137

If so perhaps we should modify it there instead of one specific route?

Reply:
```python
    @flask_app.context_processor
    def inject_service_name():
        fund_title, fund_name = FundMethods.get_service_name()
        if fund_title:
            service_title = gettext("Apply for") + " " + fund_title
        elif return_app := request.args.get("return_app"):
            service_title = Config.SAFE_RETURN_APPS[return_app].service_title
        else: 
            service_title = "Access Funding"
        return dict(
            service_title=service_title,
            fund_name=fund_name,
        )
```

Reply:
Nice one - this looks like a better idea.

Good shout on the naming.

The reason I didn't use `service_title` explicitly was because it would then override the value set globally even if it there was no return app. This seemed cleaner than having two calls to `render_template` in an `if`/`else`. Unless I'm missing a better solution?

Reply:
I'll implement the `inject_service_name` change 👍

Reply:
> Nice one - this looks like a better idea.
> 
> Good shout on the naming.
> 
> The reason I didn't use `service_title` explicitly was because it would then override the value set globally even if it there was no return app. This seemed cleaner than having two calls to `render_template` in an `if`/`else`. Unless I'm missing a better solution?

Ah that makes sense!  Thanks, nice work